STX.EFxceptions.MySql.Base.Tests.Unit/Services/Foundations/MySqlEFxceptionServiceTests.Exceptions.cs

```diff
@@ -49,7 +49,7 @@ public void ShouldThrowInvalidColumnNameException()
                     .Returns(sqlInvalidColumnNameErrorCode);
 
             // when . then
-            Assert.Throws<InvalidColumnNameMySqlException>(() =>
+            Assert.Throws<InvalidColumnNameException>(() =>
```

Comment:
This test does not seem correct.  I can't see any of the local exceptions.

```cs
        private void ConvertAndThrowMeaningfulException(int sqlErrorCode, string message)
        {
            switch (sqlErrorCode)
            {
                case 207:
                    throw new InvalidColumnNameMySqlException(message);
                case 208:
                    throw new InvalidObjectNameMySqlException(message);
                case 547:
                    throw new ForeignKeyConstraintConflictMySqlException(message);
                case 2601:
                    throw new DuplicateKeyWithUniqueIndexMySqlException(message);
                case 2627:
                    throw new DuplicateKeyMySqlException(message);
            }
        }
```

DbUpdateException => MySqlException => local exception => catagorised exception.   

Please revisit all tests on this file